scripts/calc_elf_addresses.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+
+import argparse
+from elftools.elf.elffile import ELFFile
+
+args = None
+def parse_args():
+	global args
+
+	parser = argparse.ArgumentParser(description='Helper utility to generate a linker command'
+                                         ' line with calculated ELF section addresses')
+
+	parser.add_argument("-f", "--file", required=True, type=str,
+						help='Object file name')
+	parser.add_argument("-t", "--text-addr", required=True, type=str,
+						help='.text section address')
+
+	args = parser.parse_args()
+
+def main():
+	parse_args()
+
+	elf = ELFFile(open(args.file, 'rb'))
+
+	text_addr = int(args.text_addr.replace("0x", ""), 16)
```

Comment:
Don't restrict the input to base 16, use auto-base 0:

```suggestion
	text_addr = int(args.text_addr, 0)
```

```python
>>> int("0x10",0)
16
>>> int("10",0)
10
>>> 
```